### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S subprojects,po,builddir`

Not familiar on how to make changes to `po` folder, hence skip it.

### CODE REVIEW

**Comment by rafaelmardojai** [comment]
Oh bad timing 😅. Sorry but yesterday I committed the same at #288 along with a CI job.

File: dialect/lang_selector.py [hunk 1]
```diff
@@ -54,8 +54,8 @@ def __init__(self, **kwargs):
         self.lang_model = Gio.ListStore.new(LangObject)
         self.filter = Gtk.CustomFilter()
         self.filter.set_filter_func(self._filter_func)
-        fitler_model = Gtk.FilterListModel.new(self.lang_model, self.filter)
-        self.lang_list.bind_model(fitler_model, self._create_lang_row)
+        filter_model = Gtk.FilterListModel.new(self.lang_model, self.filter)
+        self.lang_list.bind_model(filter_model, self._create_lang_row)
```
Comment (kianmeng) [inline dialect/lang_selector.py:58]: @rafaelmardojai I believe this one was not found by `codespell`. Should I close this PR?

Comment (rafaelmardojai) [reply dialect/lang_selector.py:58]: Oh, did you fixed this manually? Will merge this then.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**